File-Formats/PDF/Working-with-Annotations.md

```diff
@@ -5098,6 +5098,226 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("polygonAnnotation.pdf"
 
 {% endtabs %}
 
+### Watermark Annotation
+
+A watermark annotation is used to represent graphics that are expected to be printed at a fixed size and position on a page, regardless of the dimensions of the printed page., [PdfWatermarkAnnotation](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Interactive.PdfWatermarkAnnotation.html) can be used.
+
+The following code example explains how to add a watermark annotation in the PDF document
+
+* C#
```

Comment:
Remove this unwanted lines, the tabs will be added based on the highlight header.